backend/tests/core/ServiceHelper.cs

```diff
@@ -98,6 +105,25 @@ public static class ServiceHelper
 
             return service;
         }
+
+        /// <summary>
+        /// Creates an instance of a service of the specified 'T' type and initializes it with the specified 'user'.
+        /// Will use any 'args' passed in instead of generating defaults.
+        /// Once you create a service you can no longer add to the services collection.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="helper"></param>
+        /// <param name="args"></param>
+        /// <returns></returns>
+        public static T CreateService<T>(this TestHelper helper, params object[] args) where T : IService
```

Comment:
It will use the PimsContext that exists in the service provider instead of creating a new one.  Reduces amount of boilerplate code.